.github/workflows/main.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - fix_cross_compilation
```

Comment:
This isn't needed. There's no way to get your CI to run without our manual approval, annoyingly.